Nodejs/Product/Nodejs/NodejsFileExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿//*********************************************************//
+//    Copyright (c) Microsoft. All rights reserved.
+//    
+//    Apache 2.0 License
+//    
+//    You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//    
+//    Unless required by applicable law or agreed to in writing, software 
+//    distributed under the License is distributed on an "AS IS" BASIS, 
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or 
+//    implied. See the License for the specific language governing 
+//    permissions and limitations under the License.
+//
+//*********************************************************//
+
+using System;
+using System.IO;
+
+namespace Microsoft.NodejsTools {
+    static class NodejsFileExtensions {
```

Comment:
Can you change the name of this to be something like NodejsFileTypeHelpers? Files named with "Extensions" tend to be extension methods, so it's a bit overloaded.